packages/experiments/src/components/Announced/examples/Announced.BulkLongRunning.Example.tsx

```diff
@@ -0,0 +1,192 @@
+import * as React from 'react';
```

Comment:
On that note, we should see if we can improve the accessibility of these examples before moving the component out of experiments. It might still be worth having for testing via mouse for local development.

I imagine some of the examples evolving into their own components if their usage is common enough. The photos example comes to mind.

Reply:
Good point! I agree that these are not actions that visually impaired users would perform. The idea here was to represent the "Copying, uploading, or moving many items" scenario. I suppose that a keyboard-events-only version could be illustrated with copying and pasting several items. I'll think more about how we could show this in the demo page.